mx.h

```diff
@@ -212,13 +250,20 @@ struct MxOps
    * @param msg Message to close
    * @retval  0 Success
    * @retval -1 Failure
+   *
+   * **Contract**
+   * - @a m   is not NULL
+   * - @a msg is not NULL
```

Comment:
Add `*msg` too

Reply:
hm, what's the point there? Can a valid pointer actually point to struct Message which is Null?
We aren't passing a double pointer to struct Message, but only a struct Message*

Reply:
Sorry, ignore that.
We're only **closing** the Message handle, not freeing it.